### ISSUE TITLE
Setup DB users in better way

### ISSUE THREAD

**Issue by apeksharma** [hashgraph/hedera-mirror-node#433]
From https://github.com/hashgraph/hedera-mirror-node/pull/429#pullrequestreview-327736413

### Importer Creates Users
Pros:
- Setup of new GRPC component is automated
- Similar to how REST API user is created

Cons:
- Importer has to have knowledge of every other component's passwords/usernames
- Duplication of config properties (once #416 is complete)
- Importer has to have elevated privileges to create users
- Not similar to how importer user is created
- Requires importer be started before all other components and all patches to complete. If a patch takes 3 hours like in production yesterday, no other component can startup
- Flyway will fail and app won't start up if user already exists (e.g. if resetting db & cloud sql can't delete users, manually created, etc.)
- Can't reuse users for multiple databases on same host

### Users created outside of Importer
Pros:
- Importer doesn't need to know every other component's passwords/usernames
- Importer can have reduced privileges
- No duplicated config properties
- All application users created similarly via one mechanism
- No startup dependency
- Could be automated via docker-compose, kubernetes, scripts, etc.
- We can't delete users on cloudsql, which means that in above case, when wiping the database you then have to rename the mirror_api user to deleteme or something and then run the flyway scripts to create the schema.
- We're also going to be running several databases on the same host - the migration scripts as they currently (above case) are forces us to use different users for each database rather than having that be optional (which is also useful for local development).

Cons:
- Not automated via flyway
- Create user command doesn't have if not exists functionality

**Comment by mike-burrage-hedera** [comment]
This is a duplicate of #140
